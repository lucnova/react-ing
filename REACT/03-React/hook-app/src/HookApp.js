import React from 'react'

export const HookApp = () => {
    return (
        <div>
            <h1>Wenas</h1>
        </div>
    )
}
