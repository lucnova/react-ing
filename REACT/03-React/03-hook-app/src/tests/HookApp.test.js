import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('Testing on <HookApp />', () => {
	test('should display accordingly', () => {
		const wrapper = shallow(<HookApp />);

		expect(wrapper).toMatchSnapshot();
	});
});
