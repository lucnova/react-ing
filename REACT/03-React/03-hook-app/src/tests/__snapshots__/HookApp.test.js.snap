// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing on <HookApp /> should display accordingly 1`] = `
<div
  className="row"
>
  <div
    className="col text-center"
  >
    <h1>
      HooksApp
    </h1>
  </div>
</div>
`;
